app/main.py

```diff
@@ -1,8 +1,52 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def rate_service(self, rate: int):
+        self.average_rating = round(
+            ((self.average_rating * self.count_of_ratings) + rate)
+            / (self.count_of_ratings + 1), 1
+        )
+        self.count_of_ratings += 1
+        return self.average_rating, self.count_of_ratings
+
+    def wash_single_car(self, car: Car):
+        if self.clean_power > car.clean_mark:
+            car.clean_mark = self.clean_power
+        return car.clean_mark
+
+    def calculate_washing_price(self, car: Car):
+        dif_betw_cl_pow_and_cl_mark = self.clean_power - car.clean_mark
```

Comment:
it's a bad code style. Create temporal variables with good names with intermediate calculations